addon/utils/get-position.js

```diff
@@ -17,8 +17,8 @@ export default function getPosition(el) {
   // See https://github.com/twbs/bootstrap/issues/20280
   let rect = el.getBoundingClientRect();
   let elOffset = isBody ? { top: 0, left: 0 } : (isSvg ? {} : {
-    top: rect.top + document.body.scrollTop,
-    left: rect.left + document.body.scrollLeft
+    top: rect.top + (window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0),
```

Comment:
might be a good idea... two lines below scroll-position is also evaluated. But I don't know where it's used...